MLOps/intro-to-ml-model-deployment-and-management/intro-to-ml-model-deployment-and-management.md

```diff
@@ -0,0 +1,1010 @@
+    - Scorer: scorer metric used for this model
+    - Score: validation score using this model
+    - Owner: user who deployed this model
+    - Environment: current environment this model is deployed to
+    - Tag: tag name and value associated with this model deployment
+- Save and deploy
+
+4\. Press the **X** button in the top right corner to exit that **Deployment** page for **hydraulic_model_1**.
+
+### Interactive Scoring via Dev Deployment Environment
+
+1\. Click on **Actions 3 dot** for **hydraulic_model_1 model deployments**, then click **Show sample request**:
+
+![model-deployment-show-sample-request](./assets/model-deployment-show-sample-request.jpg)
+
+**Figure:** Model Deployment Show Sample Request
+
+Click **Copy request** button.
+
+Click **Close** button.
+
+Paste **Sample Request** into your terminal:
+
+![sample-request-result](./assets/sample-request-result.jpg)
+
+**Figure:** Model Deployment Sample Request Result
+
+### Batch Scoring via Dev Deployment Environment
+
+From Driverless AI, we insert live code for Data Recipe on our hydraulic_test data and send it as a score request to MLOps Model deployed in Dev environment.
+
+1\. Go to **Driverless AI** tab
+
+2\. Go to Driverless AI **Datasets** page
+
+3\. Click on **hydraulic_test_data**
+
+4\. Click **Details**
+
+5\. Click **Modify by Recipe** and then click **Live Code**
+
+![dai-test-data-recipe-live-code](./assets/dai-test-data-recipe-live-code.jpg)
+
+6\. Erase the previous live code template
+
+![dai-test-data-initial-live-code](./assets/dai-test-data-initial-live-code.jpg)
+
+7\. Copy the following **Data Pumper Python code** into Driverless AI **live code** editor:
+
+8\. Update the `scoring_endpoint` variable value with your particular scoring_endpoint for your current deployed model.
+
+~~~python
+import csv
+import requests
+import time
+import datatable
+from typing import List
+
+# Pump data to MLOps
+
+# X is your input dt.Frame (https://github.com/h2oai/datatable)
+X
+
+# This endpoint will need to be updated by copying your endpoint URL from current deployed model 
+scoring_endpoint = "http://34.217.40.39:1080/85bfcf27-fbbb-4248-aa1d-49796c4bc328/model/score"
+
+num_pumps = 1
+
+X_pd = X.to_pandas()
+
+for i in range(num_pumps):
+    for row in range(X.nrows):
+        print(list(X.names))
+        print(list(list(X_pd.iloc[row].values)))
+        response = requests.post(
+            url = scoring_endpoint,
+            json = {"fields": list(X.names), "rows": [[str(num) for num in list(X_pd.iloc[0].values)]]},
+        )
+        time.sleep(1)
+
+return X  # return dt.Frame, pd.DataFrame, np.ndarray or a list or named dict of those
+~~~
+
+![dai-recipe-live-code-data-pumper](./assets/dai-recipe-live-code-data-pumper.jpg)
+
+Alternatively, you could run a Python script from your CLI to execute the data pumper. Refer to the script file: [data-pumper-called-from-cli.py](./recipes/data-pumper-called-from-cli.py)
```

Comment:
While the offline script is useful, it would require a user download the dataset, which requires more steps to make sure they get it right.  If we're not providing a how-to for this use-case, would it make sense to leave this option out?

Reply:
@orendain I agree with you. So, I removed the comment on using an alternative python script.